### ISSUE TITLE
How to browse the DB Contents

### ISSUE THREAD

**Issue by garces-lb** [jepiqueau/jeep-sqlite#45]
I would just like to ask is there is a known method to see the contents of the database.

In the browser developer tools, `Application > IndexedDB > jeepSqliteStore > databases` I can see my database(s) as a `UIntArray`. I guess if I manage to download the binary I could open it in a SQLite Browser. And maybe with some plugin or tool I could explore the DB contents in the browser itself. I don't know how to download the binary or any plugins to browse it in place. Is there any way to do it?

**Comment by jepiqueau** [comment]
@garces-lb well use the saveToLocalDisk method and open it with DB-browser or any other database sqlite browser.

**Comment by jepiqueau** [comment]
@garces-lb 
As i do not get get any comments i assume my answer suit your need, so i close the issue.

**Comment by garces-lb** [comment]
Hi. Sorry I did not answer, I did not want to fill this thread with (probably) useless stuff. I guess your solution would be valid for most users, but not for me, as I do not reach any `saveToLocalDisk` function, neither in `SQLiteServier` nor in `SQLiteDBConnection`. I guess this function was included in later versions than the one I use. I am using `2.6.2` of `jeep-sqlite` and `1.10.2` in `sql.js`.

Trying to update those has usually brought me problems on compatibility, as my project will be deployed in iOS, Android, Electron and Browser. I will try it more thoroughly when I find the time, maybe then I get the `saveToLocalDisk` function working.
